releases/release-1.29/release-notes/maps/pr-119346-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 119346
+releasenote:
+  text: |-
+    `kubeadm`: printing the default component configs for `reset` and `join` is now
+    unsupported.
```

Same as [previous comment](https://github.com/kubernetes/sig-release/pull/2347/files#r1339036210).

```suggestion
    '`kubeadm`: printing the default component configs for `reset` and `join` is now
    unsupported.'
```